### DISCUSSION TITLE
Additional placeholders and/or subscriber fields?

### DISCUSSION THREAD

**Discussion by michaelzap** [mettle/sendportal#198] [Help]
Hello -

We routinely need to send out bulk emails that include more personalized info than just the subscriber's name and email. For example, we might want to thank someone for their 2021 donations totaling X amount or we might want to have a {{greeting}} placeholder for when first and last name would not be appropriate. How can we extend SP to do this sort of thing?

We would also like to be able to script placeholders into the {{content}} contents. For example, we might want to include a personalized URL within the email content. In other cases, we might want to output one snippet of text if a subscriber is tagged as a monthly donor and a different snippet if not. Up till now we have been using a custom system that I threw together which can do these sorts of things but is inferior in pretty much every other way to SP.

Do you have any documentation, code, or pointers for adding such functionality?

Thanks in advance for your reply!

**Comment by inukurax** [comment]
I was looking for the same.
Having multiple text blocks in my template, would so having content1, content2 and so on would help.
